pkg/cmd/subscription/get/get.go

```diff
@@ -0,0 +1,91 @@
+package get
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type GetOptions struct {
+	HttpClient func() (*http.Client, error)
+	BaseRepo   func() (ghrepo.Interface, error)
+	IO         *iostreams.IOStreams
+
+	Repository string
+}
+
+func NewCmdGet(f *cmdutil.Factory, runF func(*GetOptions) error) *cobra.Command {
+	opts := &GetOptions{
+		IO:         f.IOStreams,
+		HttpClient: f.HttpClient,
+		BaseRepo:   f.BaseRepo,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Get subscription information to the repository",
+		Long: heredoc.Doc(`
+			Get subscription information to the repository.
+
+			Without any arguments, this command prints out the subscription information
+			for the current repository.
+
+			The possible subscription states are:
+			- all: you are watching the repository and notified of all notifications.
+			- ignore: you are watching the repository but not notified of any notifications.
+			- unwatched: you are not watching the repository or custom notification is set.
+		`),
+		Example: heredoc.Doc(`
+			$ gh subscription get
+			$ gh subscription get monalisa/hello-world
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				opts.Repository = args[0]
+			}
+
+			if runF != nil {
+				return runF(opts)
+			}
+
+			return getRun(opts)
```

Comment:
Keeping code more tightly together, nit Go practice

```suggestion
			if runF != nil {
				return runF(opts)
			}
			return getRun(opts)
```